apps/consent-ui/src/components/layouts/SideImageLayout/index.tsx

```diff
@@ -0,0 +1,109 @@
+ * EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES
+ * OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT
+ * SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+ * INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED
+ * TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS;
+ * OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER
+ * IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import Image, { StaticImageData } from 'next/image';
+import clsx from 'clsx';
+import { ReactNode } from 'react';
+import Link from 'next/link';
+
+import OICRLogoEN from 'src/public/oicr-logo-gray-en.svg';
+import ChevronSvg from 'src/public/chevron-large.svg';
+import { ValidLanguage, getTranslation } from 'src/i18n';
+import LanguageToggle from 'src/components/common/Header/LanguageToggle';
+import HelpButton from 'src/components/common/Header/HelpButton';
+import LocalizedLink from 'src/components/common/Link/LocalizedLink';
+import { RouteName } from 'src/components/common/Link/types';
+
+import styles from './SideImageLayout.module.scss';
+
+interface SideImageLayoutProps {
+	children: ReactNode;
+	className?: string;
+	currentLang: ValidLanguage;
+	desktopHeaderImage: StaticImageData;
+	desktopNavAction?: { bottomText: string; topText: string; url: RouteName };
+	mainSubtitle: ReactNode;
+	mainTitle: string;
+	navTitle: string;
+}
+
+const SideImageLayout = ({
+	children,
+	className,
+	currentLang,
+	desktopHeaderImage,
+	desktopNavAction,
+	mainSubtitle,
+	mainTitle,
+	navTitle,
+}: SideImageLayoutProps) => {
+	const translate = getTranslation(currentLang);
+
+	return (
+		<div className={clsx(styles.container, className)}>
+			<header className={styles.desktopHeader}>
+				<Image className={styles.image} src={desktopHeaderImage} alt="" />
+				<div className={clsx(styles.content)}>
+					<Link href={`/${currentLang}`} className={styles.logoLink}>
+						<Image
+							src={OICRLogoEN}
+							alt={translate('footer', 'oicrLogoAlt')}
+							className={styles.logoImg}
+						/>
+					</Link>
+					<h1 className={styles.title}>{navTitle}</h1>
+				</div>
+			</header>
+			<header className={styles.mobileTabletHeader}>
+				<h1 className={styles.title}>{navTitle}</h1>
+			</header>
+			<div className={styles.main}>
+				<nav className={styles.desktopNav}>
+					<div className={styles.leftButtons}>
+						<LanguageToggle currentLang={currentLang} />
+						<HelpButton label={translate('header', 'help')} />
+					</div>
+					<div className={styles.rightButtons}>
+						{desktopNavAction && (
+							<LocalizedLink
+								className={styles.desktopNavAction}
```

Comment:
```suggestion
								className={styles.desktopNavAction}
								defaultStyle={false}
```
In #274 I updated the LocalizedLink component so it has the link styles by default (I got sick of adding those same four css lines everywhere) so I think you'll have to disable that here, this is what I'm seeing right now:
<img width="291" alt="image" src="https://github.com/OHCRN/platform/assets/70766739/d7d050a2-d971-4cd0-bdc0-2b095072b72d">
But setting `defaultStyle={false}` should disable the styles and leave it as it was.
<img width="284" alt="image" src="https://github.com/OHCRN/platform/assets/70766739/d107f24a-426f-4571-8ade-4dbd85f9a2cd">

Reply:
Please, disregard if I'm being unreasonably pedantic: I think the props name `defaultStyle` indicates and expectation to be told "'which' style will be default?", not that its value is meant as an "on/off" boolean.

Instead, considering that the intent here is to "do something", then the value passed should be truthy and therefor `disableDefaultStyle = true` would be more declarative.

Looking forward to hearing others' thoughts

Reply:
i agree with you but this was an existing change prior to my PR, we should make a separate ticket/PR to discuss

Reply:
https://github.com/OHCRN/platform/issues/341